core/types/calls.cc

```diff
@@ -518,6 +518,17 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, const DispatchArgs &arg
 
     SymbolRef mayBeOverloaded = symbol.data(gs)->findMemberTransitive(gs, args.name);
 
+    if (!mayBeOverloaded.exists()) {
+        // Before raising any error, we look if the method exists in all required ancestors by this symbol
+        auto ancestors = symbol.data(gs)->requiredAncestorsTransitive(gs);
```

Comment:
Only is the method is not found, this part is where we pay the most perf:

We look into all transitive superclasses (recursive like when we look with `findMemberTransitive`) and in the linearized list of mixins if they require an ancestor.